workflows/update-version.yml

```
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Canonical Ltd.
+name: Update Version
+on:
+  workflow_call:
+    inputs:
+      branch-name:
+        description: Name of the branch to checkout
+        required: false
+        type: string
+        default: ${{ github.ref }}
+      release-changed:
+        description: Name of the branch to checkout
+        required: false
+        type: boolean
+      release-version:
+        description: Name of the branch to checkout
+        required: false
+        type: boolean
+
+
+jobs:
+  update-version:
+    runs-on: ubuntu-latest
+    if: ${{ input.release-changed == 'true' }}
```

Given that you are defining this variable as `boolean`, do you need to indicate `true` as a string?
```suggestion
    if: ${{ inputs.release-changed == 'true' }}
    runs-on: ubuntu-latest
```

---

No. Fixed.